misc/integrated_game_data.sql

```diff
@@ -1,24 +1,52 @@
+/* Create languages */
+INSERT INTO languages
+	(id, created_at, name, abbr)
+VALUES
+	(1, '01/01/2000', 'English', 'en'),
+	(2, '01/01/2000', 'Macedonian', 'mk'),
+	(3, '01/01/2000', 'Albanian', 'al'),
+	(4, '01/01/2000', 'Turkish', 'tr');
```

Comment:
This should already be added in the data migrations. You also don't need to create the `id` or `created_at` fields, they are auto-populated.

Reply:
Actually I managed to run it! Not having that table in the db did not allow me to create the other tables